### DISCUSSION TITLE
What version of Anaconda is this compatible with?

### DISCUSSION THREAD

**Discussion by PAFlores** [oracle/python-oracledb#80] [Q&A] [answered]
I am a ta client that has a very strict security model and thus we are on Citrix VDI's.  Have been trying to get oracledb installed (conda install -c conda-forge oracledb) and I am seeing several conflicts, to numerous to include here.  Which raises the question as to what version of anaconda  should I ask the client to provide? 
This is what I see in conda list:  anaconda    version: 2021.05   build.py38_0

**Comment by anthony-tuininga** [comment, marked as answer]
That sounds like a question for Anaconda. I am not very familiar with Anaconda and did not write the recipe that installs python-oracledb, so not sure how I can help!
